### DISCUSSION TITLE
run example ‘fox’ with bad result

### DISCUSSION THREAD

**Discussion by benben516252275** [NVlabs/instant-ngp#799] [Q&A]
hi：
   I run instant-ngp in docker. I run example 'fox' in data/nerf/fox with command “python3.7 ./scripts/run.py --mode nerf --scene ./data/nerf/fox/ --save_snapshot saved/fox.msgpack --save_mesh saved/fox.ply  --train”。
  when i open fox.ply in meshlab, The rendering of 3D model is very fuzzy。How can I solve this problem? thank you.
![image](https://user-images.githubusercontent.com/48997706/179974927-887c2505-b5dd-4c5a-80e8-39faad977527.png)

**Comment by Tom94** [comment]
__instant-ngp__ doesn't lend itself to mesh generation. The bad-quality mesh you're seeing is unfortunately expected when meshing real-world scenes.

**Reply by benben516252275** [reply]
Hi export：
   Does that mean i can only open it in testbed-gui for high-quality render?
